.github/workflows/run-daily-aggre.yaml

```
@@ -0,0 +1,105 @@
+  # - cron: "*/30 * * * *"
+
+jobs:
+  scrape-velog-posts:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+
+      - name: Install Python
+        uses: actions/setup-python@v4
```

actionlint (1.7.4)

the runner of "actions/setup-python@v4" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)